IDE/MonoDevelop/MonoDevelop.MonoGame.iOS/Properties/AssemblyInfo.cs

```diff
@@ -1,26 +1,22 @@
 using System.Reflection;
 using System.Runtime.CompilerServices;
 
-// Information about this assembly is defined by the following attributes. 
+// Information about this assembly is defined by the following attributes.
 // Change them to the values specific to your project.
-
-[assembly: AssemblyTitle("${ProjectName}")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("${AuthorCompany}")]
-[assembly: AssemblyProduct("")]
-[assembly: AssemblyCopyright("${AuthorCopyright}")]
-[assembly: AssemblyTrademark("${AuthorTrademark}")]
-[assembly: AssemblyCulture("")]
-
+[assembly: AssemblyTitle ("MonoDevelop.MonoGame.iOS")]
+[assembly: AssemblyDescription ("")]
+[assembly: AssemblyConfiguration ("")]
+[assembly: AssemblyCompany ("")]
+[assembly: AssemblyProduct ("")]
+[assembly: AssemblyCopyright ("Dean")]
```

Comment:
Is this the assembly info for the plug-in or the template for new projects?  If for the plug-in, the copyright should be "MonoGame Team" like our standard file header.  If for a template, it should be the token so the IDE fills it in as appropriate for the user.